.travis.yml

```diff
@@ -21,7 +21,8 @@ before_script:
 
 script:
 - |
-    travis-cargo build && travis-cargo test
+    travis-cargo build && travis-cargo test &&
+    travis-cargo build -- --features tls && travis-cargo test -- --features tls
 
 after_success:
 - travis-cargo coveralls --no-sudo
```

Comment:
These run just like `cargo build --features tls`. If you pass `--` to `travis-cargo` then whatever follows is passed to `cargo`.